gnocchi/service.py

```diff
@@ -85,9 +102,6 @@ def prepare_service(args=None, conf=None,
                                                 'rest', 'policy.json'))
     conf.set_default('policy_file', cfg_path, group='oslo_policy')
 
-    log.set_defaults(default_log_levels=log.get_default_log_levels() +
-                     ["passlib.utils.compat=INFO"])
```

Comment:
just curious, are we still able to control the logging level of our dependencies? ie we only want ERROR from dependencies?